pkg/kubelet/status/status_manager.go

```diff
@@ -156,10 +156,10 @@ func (m *manager) Start() {
 	}
 
 	klog.InfoS("Starting to sync pod status with apiserver")
-	//lint:ignore SA1015 Ticker can link since this is only called once and doesn't handle termination.
-	syncTicker := time.Tick(syncPeriod)
+	syncTicker := time.NewTicker(syncPeriod)
 	// syncPod and syncBatch share the same go routine to avoid sync races.
 	go wait.Forever(func() {
+		defer syncTicker.Stop()
```

Comment:
calling wait.Forever adds a panic recovery path that can re-call the provided function. If that ever occurs, this change will break the code because it will have stopped the ticker after the function exited the first time

Reply:
@liggitt  Got it, thanks. Should I move the `syncTicker := time.NewTicker(syncPeriod)` into the wait.Forever function then so that a new ticker is created if the function is re-called? Or what would you recommend here?

Reply:
no, I would keep the lint:ignore directive